gift-mode.el

```diff
@@ -42,6 +42,50 @@
 (require 'outline)
 (require 'newcomment)
 
+
+(defvar gift-imenu-comments-regexp "\\(\\(^\\s-*//.*$\\)\\|\\(^\\s-*$\\)\\)+")
+
+(defvar gift-imenu-question-regexp "^\\(\\([^{}]\\)+\\){" )
+
+(defvar gift-imenu-title-regexp "::\\(.*\\)::" )
```

Comment:
Similarly, we should probably detect escaped colons here: `::\\(\\(\\\\:\\|[^:]\\)*\\)::`, no?

Reply:
In my last version, this regex is used after unescape the title, so backslashes can be ignored.